applications/asset_tracker_v2/src/ext_sensors/ext_sensors.c

```diff
@@ -101,16 +91,14 @@ static void accelerometer_trigger_handler(const struct device *dev,
 		evt.value_array[1] = sensor_value_to_double(&data[1]);
 		evt.value_array[2] = sensor_value_to_double(&data[2]);
 
-		/* Do a soft filter here to avoid sending data triggered by
-		 * the inactivity threshold.
-		 */
-		if ((abs(evt.value_array[0]) > threshold ||
-		     (abs(evt.value_array[1]) > threshold) ||
-		     (abs(evt.value_array[2]) > threshold))) {
-
-			evt.type = EXT_SENSOR_EVT_ACCELEROMETER_TRIGGER;
-			evt_handler(&evt);
+		if (trig->type == SENSOR_TRIG_MOTION) {
+			evt.type = EXT_SENSOR_EVT_ACCELEROMETER_ACT_TRIGGER;
+			LOG_DBG("Acceleration detected");
```

Comment:
```suggestion
			LOG_DBG("Activity detected");
```

Reply:
Sorry, the wording should probably be `activity` instead of `acceleration` for alignment reasons. Keep the wording consistent.